modules/installation-about-mirror-registry.adoc

```diff
@@ -7,18 +7,19 @@
 [id="installation-about-mirror-registry_{context}"]
 = About the mirror registry
 
-You can mirror the images that are required for {product-title} installation and subsequent product updates to a mirror registry. These actions use the same process. The release image, which contains the description of the content, and the images it references are all mirrored. In addition, the Operator catalog source image and the images that it references must be mirrored for each Operator that you use. After you mirror the content, you configure each cluster to retrieve this content from your mirror registry.
+You can mirror the images that are required for {product-title} installation and subsequent product updates to a container mirror registry such as Red Hat Quay, JFrog Artifactory, Sonatype Nexus Repository, or Harbor. If you do not have access to a large-scale container registry, you can use the _mirror registry for Red Hat Openshift_, a small-scale container registry included with all {product-title} subscriptions.
 
-The mirror registry can be any container registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2]. All major cloud provider registries, as well as Red Hat Quay, Artifactory, and others, have the necessary support. Using one of these registries ensures that {product-title} can verify the integrity of each image in disconnected environments.
+Whether using Red Hat Quay, Artifactory, Sonatype Nexus Repository, Harbor, or the mirror registry for Red Hat Openshift, all actions use the same process. The release image, which contains the description of the content, and the images it references are all mirrored. In addition, the Operator catalog source image and the images that it references must be mirrored for each Operator that you use. After you mirror the content, you configure each cluster to retrieve this content from your mirror registry.
```

Comment:
Order these with Red Hat products being first. 

1. Quay
2. Quay (OMR)
3. Artifactory
4. Sonatype Nexus
5. Harbor

---

What are 'actions'? I get that what you are trying to say; but I think it could be framed better. 

IE: No matter what 'image registry' you choose to use; the procedure to mirror content from Red Hat hosted sites (on the internet), to an isolated image registry, is the same.